### ISSUE TITLE
Clarification on training process

### ISSUE THREAD

**Issue by koamd** [yoonkicho/PPLR#18]
The training process in [#L233](https://github.com/yoonkicho/PPLR/blob/5337193c4a2c7d0870c558a37d28634744d5b6e5/examples/train_pplr.py#L233) looks like it is using a labelled dataset, but in the paper, it mentions that all of the training are done using pseudo labels. 

If I am to train the model using an unsupervised dataset, I would have to specifically modify all the targets to be pseudo labels correct?

**Comment by koamd** [comment]
Made a mistake in reading the code. Realized that train_loader is loading the dataset with the pseudo labels.
